tendrl/gluster_integration/flows/brick_mapping/__init__.py

```diff
@@ -0,0 +1,22 @@
+from tendrl.commons import flows
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+# This flow will be invoked only after cluster is available
+# in the tendrl (either by creation/import)
+class BrickMapping(flows.BaseFlow):
```

Comment:
I feel flow names should start with verb so it could something like `GenerateBrickMapping`